src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -232,10 +234,24 @@ export class MarkdownToolbarComponent extends AngularDisposable {
 			if (this.cellModel.currentMode !== CellEditModes.WYSIWYG) {
 				needsTransform = false;
 			} else {
+				let result = undefined;
+				const isFile = !URI.parse(linkCalloutResult?.insertUnescapedLinkUrl).scheme.includes('http');
+				if (!path.isAbsolute(linkCalloutResult?.insertUnescapedLinkUrl) && isFile) {
+					let midDirectories = (linkCalloutResult?.insertUnescapedLinkUrl.match(/..\\/g) || []).length;
```

Comment:
comment would be good here. it's not immediately clear what `midDirectories` refers to.

Reply:
Can we change this to include the string `num` in the variable name somewhere? For clarity 😄

Reply:
using fs.resolve instead